signac_dashboard/dashboard.py

```diff
@@ -223,11 +223,12 @@ def run(self, *args, **kwargs):
         """
         host = self.config.get("HOST", "localhost")
         port = self.config.get("PORT", 8888)
+        debug = self.config.get("DEBUG", False)
         max_retries = 5
 
         for _ in range(max_retries):
             try:
-                self.app.run(host, port, *args, **kwargs)
+                self.app.run(host=host, port=port, debug=debug, *args, **kwargs)
```

Comment:
It is not allowed to specify `*args` after keyword arguments. I think this will raise an error if `*args` is not empty.